draft-ietf-ppm-dap.md

```diff
@@ -974,15 +982,14 @@ are supported by a participant is outside of this document's scope.
 
 ## Errors
 
-Errors can be reported in DAP both as HTTP status codes and as problem detail
-objects {{!RFC9457}} in the response body. For example, if the HTTP client sends
-a request using a method not allowed in this document, then the server MAY
-return HTTP status 405 Method Not Allowed.
+Errors are reported as HTTP status codes. Any of the standard client or server
+errors (the 4xx or 5xx classes, respectively, from {{Section 15 of !RFC9110}})
+are permitted.
 
 When the server responds with an error status code, it SHOULD provide additional
-information using a problem detail object. If the response body does consist of
-a problem detail object, the HTTP status code MUST indicate a client or server
-error (the 4xx or 5xx classes, respectively, from {{Section 15 of !RFC9110}}).
+information using a problem detail object {{!RFC9457}} in the response body. If
+the response body does consist of a problem detail object, the status code MUST
+indicate a client or server error.
```

Comment:
Unrelated to this PR, but I'm confused by "client or server error". What do we mean by this?

Reply:
HTTP 4xx or 5xx. This is explained in the paragraph above with a reference to RFC 9110.